packages/atomic/src/components/atomic-query-summary/atomic-query-summary.tsx

```diff
@@ -39,30 +30,11 @@ interface Summary {
 export class AtomicQuerySummary implements InitializableComponent {
   @InitializeBindings() public bindings!: Bindings;
   public querySummary!: QuerySummary;
-  private unescapeStringOption = {interpolation: {escapeValue: false}};
 
   @BindStateToController('querySummary')
   @State()
   private querySummaryState!: QuerySummaryState;
-  @BindStateToI18n()
```

Comment:
Used this component as an example of what we can do now with this change. There is way less boilerplate, BindStateToI18n is no more necessary and we can use the bindings.i18n.t method directly.

If this is accepted, I can go ahead and refresh all the components in the future.

Reply:
As an additional note, it's only deprecated but the "old way" still works perfectly